README.md

```diff
@@ -31,14 +31,83 @@ $ editor .env # edit your vars
 
 ## Quickstart
 
-Start the Celery worker:
+Start the Celery worker and dev server:
 
-    celery -A goodtablesio.tasks worker --loglevel=info
+```bash
+bash1$ make celery
+bash2$ make dev
+```
 
-You can use the `goodtablesio` command line helper to send validation tasks to the queue:
+Now developmet server runs on `localhost:5000`. We could send github repo for validation getting task identifiers as a response:
 
-    goodtablesio https://raw.githubusercontent.com/amercader/newcastle-libraries-data/master/councillors-wards.csv
+> localhost:5000/github/hook/
 
-or
+```json
+{
+  "repository": {
+      "clone_url": "https://github.com/roll/goodtables-example.git"
+    }
+}
+---
+693f40f0-fcad-416e-b2c7-a5beebff4f44
+```
 
-    goodtablesio  /path/to/local/file.csv
+> localhost:5000/api/task/693f40f0-fcad-416e-b2c7-a5beebff4f44
+
+```json
+{
+  "repository": {
+      "clone_url": "https://github.com/roll/goodtables-example.git"
+    }
+}
```

Comment:
Is this supposed to be a payload you send to this endpoint? This one is a GET request right without any params right? (the id is the URL)